lib/generate-id.js

```diff
@@ -22,11 +22,41 @@ const findId = async ( model, length ) => {
 }
 
 /**
- * Generates a line id and a shabad id, using the database.
+   * Logs the generated id to the console
+   * @param {Number} count the count of id to generate
+   * @param {String} type shabad, line
+   */
+const logGeneratedID = async function ( count, type ) {
+  let i = 0
+
+  if ( type === 'shabad' ) {
+    for ( i = 0; i < count; i += 1 ) {
+      console.log( `${i + 1}. New Shabad ID: `.bold.magenta + `${await findId( Shabads, 3 )}`.rainbow )
+    }
+  } else if ( type === 'line' ) {
+    for ( i = 0; i < count; i += 1 ) {
+      console.log( `${i + 1}. New Line ID: `.bold.magenta + `${await findId( Lines, 4 )}`.rainbow )
+    }
+  } else { // Default log all types
+    for ( i = 0; i < count; i += 1 ) {
+      console.log( `${i + 1}. New Shabad ID: `.bold.magenta + `${await findId( Shabads, 3 )}`.rainbow )
+      console.log( `${i + 1}. New Line ID: `.bold.cyan + `${await findId( Lines, 4 )}`.rainbow )
+    }
+  }
+}
+
+/**
+ * Interpret user command and print the id, using the database.
  */
 const main = async () => {
-  console.log( 'New Line ID: '.bold.cyan + `${await findId( Lines, 4 )}`.rainbow )
-  console.log( 'New Shabad ID: '.bold.magenta + `${await findId( Shabads, 3 )}`.rainbow )
+  commander
+    .version( '0.0.1' )
+    .description( 'Generate ID\'s for ShabadOS Database \n\nCommand \nshabad \t\tgenerate id\'s for shabads \nline \t\tgenerate id\'s for line' )
```

Comment:
A hint: you can use backticks ``` to signify multiline strings, or strings you need to escape, making it a little more natural in the code. I'd also use the the `idTypes` object mentioned above to generate a string list of things you can generate, e.g.:

`Object.keys( idTypes ).join(', ')`

Reply:
eslint: Multiline support is limited to browsers supporting ES5 only `no-multi-str`